ewasm-testnet-geth-config.json

```diff
@@ -0,0 +1,71 @@
+{
+  "config": {
+    "chainId": 66,
+    "homesteadBlock": 0,
+    "eip150Block": 0,
+    "eip150Hash": "0x0000000000000000000000000000000000000000000000000000000000000000",
+    "eip155Block": 0,
+    "eip158Block": 0,
+    "byzantiumBlock": 0,
+    "ethash": {}
+  },
+  "nonce": "0x0000000000000042",
+  "timestamp": "0x5A5B92D7",
+  "extraData": "0x655741534d2074657374206e6574776f726b2030", 
+  "gasLimit": "0x989680",
+  "difficulty": "0x400",
+  "mixHash": "0x0000000000000000000000000000000000000000000000000000000000000000",
+  "coinbase": "0x0000000000000000000000000000000000000000",
+  "alloc": {
+    "0000000000000000000000000000000000000000": {
+      "balance": "0x1"
+    },
+    "0000000000000000000000000000000000000001": {
+      "balance": "0x1",
+      "precompiled": { "name": "ecrecover", "linear": { "base": 3000, "word": 0 } }
+    },
+    "0000000000000000000000000000000000000002": {
+      "balance": "0x1",
+      "precompiled": { "name": "sha256", "linear": { "base": 60, "word": 12 } }
+    },
+    "0000000000000000000000000000000000000003": {
+      "balance": "0x1",
+      "precompiled": { "name": "ripemd160", "linear": { "base": 600, "word": 120 } }
+    },
+    "0000000000000000000000000000000000000004": {
+      "balance": "0x1",
+      "precompiled": { "name": "identity", "linear": { "base": 15, "word": 3 } } 
+    },
+    "0000000000000000000000000000000000000005": {
+      "balance": "0x1",
+      "precompiled": { "name": "modexp", "startingBlock" : "0x2dc6c0" }
+    },
+    "0000000000000000000000000000000000000006": {
+      "balance": "0x1",
+      "precompiled": { "name": "alt_bn128_G1_add", "startingBlock" : "0x2dc6c0", "linear": { "base": 500, "word": 0 } }
+    },
+    "0000000000000000000000000000000000000007": {
+      "balance": "0x1",
+      "precompiled": { "name": "alt_bn128_G1_mul", "startingBlock" : "0x2dc6c0", "linear": { "base": 40000, "word": 0 } }
+    },
+    "0000000000000000000000000000000000000008": {
+      "balance": "0x1",
+      "precompiled": { "name": "alt_bn128_pairing_product", "startingBlock" : "0x2dc6c0" }
+    },
+    "000000000000000000000000000000000000000a": {
+      "balance": "0x1",
+      "code": "0x0061736d0100000001170560017e006000017f60037f7f7f0060027f7f0060000002580408657468657265756d06757365476173000008657468657265756d0f67657443616c6c4461746153697a65000108657468657265756d0c63616c6c44617461436f7079000208657468657265756d0666696e6973680003030201040503010001071102066d656d6f72790200046d61696e00040a25012301017f1001210042ba022000ad7e42e40080100041004100200010024100200010030b"
```

Comment:
Is this the 'dummy' metering contract?  Before we merge this, we need to make sure we are using a functioning metering contract.

Reply:
Yes, it is the same: https://github.com/ewasm/testnet/blob/master/dummymetering.wasmhex

Reply:
> Before we merge this, we need to make sure we are using a functioning metering contract.

I think we should merge it as soon as possible.

The aleth config is merged with the dummy one too.

Reply:
So I should remove the parity metering contract from the testnet and replace it with the dummy one?

Reply:
because with the config in this PR you can't connect to the testnet as it is currently configured (with the parity metering contract).

Reply:
Is the aleth config is this repo up to date with testnet genesis?

if not, then I think merging this PR makes sense and then updating both later.

Reply:
Okay.